flink-table-store-core/src/main/java/org/apache/flink/table/store/file/FileStoreOptions.java

```diff
@@ -92,6 +92,17 @@ public class FileStoreOptions implements Serializable {
                             "The default partition name in case the dynamic partition"
                                     + " column value is null/empty string.");
 
+    public static final ConfigOption<Boolean> COMPACTION_RESCALE_BUCKET =
+            key("compact.rescale-bucket")
```

Comment:
> `compaction.rescale-bucket`? There are some `compaction.xxx` in `MergeTreeOptions`.

So your suggestion is to move this config key to `MergeTreeOptions`?

Reply:
ah.. my point is `compaction` instead of `compact`.
About where, maybe it is better to move this key to `TableStoreFactoryOptions`? Because actually only TableStoreFactory will read it.
(In future, maybe we should merge all `XXOptions` to one class.)